tests/CalDAV/Schedule/IMipPluginRecurrentEventTest.php

```diff
@@ -1196,4 +1196,127 @@ function testExpiredRecurrentEvent()
         $plugin->schedule($itipMessage);
         $this->assertEquals('1.1', $itipMessage->scheduleStatus);
     }
+
+    /**
+     * Test for issue #152
+     * When organizer modifies an occurrence that doesn't include an attendee,
+     * that attendee should NOT receive an iTIP notification
+     */
+    function testShouldNotSendUpdateToUninvitedAttendeesWhenOrganizerModifiesOtherInstances()
```

Comment:
We should add a test to verify iTIP REQUEST is sent when the organizer updates occurrence second to invite an attendee (recurrence event, attendee invited only for second).